salt/modules/virtualenv_mod.py

```diff
@@ -43,6 +44,39 @@ def __virtual__():
     return __virtualname__
 
 
+def virtualenv_ver(venv_bin, user=None, **kwargs):
+    '''
+    return virtualenv version if exists
+    '''
+    # Virtualenv package
+    try:
+        import virtualenv
+        version = getattr(virtualenv, '__version__', None)
+        if not version:
+            version = virtualenv.virtualenv_version
+        virtualenv_version_info = tuple(
+            [int(i) for i in version.split('rc')[0].split('.')]
+        )
+    except ImportError:
+        # Unable to import?? Let's parse the version from the console
+        version_cmd = [venv_bin, '--version']
+        ret = __salt__['cmd.run_all'](
+                version_cmd, runas=user, python_shell=False, **kwargs
+            )
+        if ret['retcode'] > 0 or not ret['stdout'].strip():
```

Comment:
For python2 output goes to `stderr`. For Python3.4+ the output goes to `stdout`. See [this comment](https://github.com/saltstack/salt/issues/56205#issuecomment-598563414) on the original issue